.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```
@@ -0,0 +1,57 @@
+name: Bug Report
+description: File a bug report.
+title: "[Bug] <title>"
+labels: ["Bug"]
```

```suggestion
type: "Bug"
```

I would prefer to use `type` field instead of labels for this.

---

The "labels" field auto adds the Bug label to the issue.

---

Yes we use a “type” field instead of labels, so would be great to auto-assign type here !

---

The issue template form is processed by Github.  I don't decide which fields are used to do what. If I include the labels field, the label will be automatically applied to the issue list view.  If I remove it, the label will not be assigned and someone will need to do that manually.

---

"type" isn't an allowed key for a Github template form.  Using "type" as the key instead of labels which is specified in the documentation, throws an error.